corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -180,7 +186,10 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                         $home.on('textchange', 'input', self.change)
                             // all select2's are represented by an input[type="hidden"]
                             .on('change', 'select, input[type="hidden"]', self.change)
-                            .on('click', 'a', self.change);
+                            // help links should not trigger a change.
+                            // I did not see anchor tags that *should* trigger a change, but left in this trigger.
+                            // Ideally, changes should be triggered off something more specific than an anchor tag
+                            .on('click', ':not(.hq-help) > a', self.change);
```

Comment:
I blamed this back to https://github.com/dimagi/commcare-hq/pull/483/ which added some links that are styled like buttons and/or icons. I imagine they could be updated to be real buttons, but understand if that's too much scope creep.

Reply:
Buttons are better semantically, but I'm not sure that changes things all that much, because clicking a button shouldn't automatically invalidate the save state either -- we don't know what that button does. For example, in the linked PR, the `addProperty` button is implemented as an anchor, although now it is a button. That button intentionally doesn't force an invalid save state, because that is meant to happen when the user selects a question or replaces the text in the new property, not when a blank property is created.
On the other hand, something like `removeProperty` also doesn't need a blanket trigger, because it explicitly calls `saveButton.fire('change')` on its own. If an action should invalidate the button, I like the explicit pattern that `removeProperty` follows.